ajax/libs/vue-lazyload/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "vue-lazyload",
+  "filename": "vue-lazyload.js",
+  "version": "1.0.6",
+  "description": "Vue module for lazy-loading images in your vue.js applications.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hilongjw/vue-lazyload.git"
+  },
+  "homepage": "https://hilongjw.github.io/vue-lazyload/",
+  "keywords": [
+    "vue-lazyload",
+    "vue",
+    "lazyload",
+    "vue-directive"
+  ],
+  "author": "Awe <hilongjw@gmail.com>",
+  "license": "MIT",
+  "npmName": "vue-lazyload",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "vue-lazyload!(*es5|*next)*"
```

Comment:
Why we need to exclude `*next*`?
Don't we need `es5.js`?
Some `vue-lazyload.js` aren't minified, we need to manually add them because we won't set auto-update minify for current filemap.